bitmap-team-TeamName/test/color-constructor-test.js

```diff
@@ -0,0 +1,141 @@
+        testObj.butterflyRed = stringifiedData;
+        done();
+      });
+    });
+    before((done) => {
+      fs.readFile(`${__dirname}/butterfly-red-test.txt`, 'hex', function(err, data){
+        if (err) throw err;
+        testObj.redControl = data;
+        done();
+      });
+    });
+    it('outputted test file should match newly transformed bitmap buffer', function(done){
+      expect(testObj.butterflyRed).to.equal(testObj.redControl);
+      done();
+    });
+  });
+  describe('Transformed file buffer should match test control - Blue Shift', function(){
+    before((done) => {
+      fs.readFile(`${__dirname}/../img/butterfly-blue.bmp`, 'hex', function(err, data) {
+        if (err) throw err;
+        let buffer = Buffer.from(data);
+        let stringifiedData = buffer.toString('hex');
+        testObj.butterflyBlue = stringifiedData;
+        done();
+      });
+    });
+    before((done) =>{
+      fs.readFile(`${__dirname}/butterfly-blue-test.txt`, 'hex', function(err, data) {
+        if (err) throw err;
+        testObj.blueControl = data;
+        done();
+      });
+    });
+    it('outputted test file should match newly transformed bitmap buffer', function(done){
+      expect(testObj.butterflyBlue).to.equal(testObj.blueControl);
+      done();
+    });
+  });
+  describe('Transformed file buffer should match test control - Purple Shift', function(){
+    before((done) => {
+      fs.readFile(`${__dirname}/../img/butterfly-purple.bmp`, 'hex', function(err, data) {
+        if (err) throw err;
+        let buffer = Buffer.from(data);
+        let stringifiedData = buffer.toString('hex');
+        testObj.butterflyPurple = stringifiedData;
+        done();
+      });
+    });
+    before((done) => {
+      fs.readFile(`${__dirname}/butterfly-purple-test.txt`, 'hex', function(err, data) {
+        if (err) throw err;
+        testObj.purpleControl = data;
+        done();
+      });
+    });
+    it('outputted test file should match newly transformed bitmap buffer', function(done){
+      expect(testObj.butterflyPurple).to.equal(testObj.purpleControl);
+      done();
+    });
+  });
+  describe('Transformed file buffer should match test control - Invert', function(){
+    before((done) => {
+      fs.readFile(`${__dirname}/../img/butterfly-invert.bmp`, 'hex', function(err, data) {
+        if (err) throw err;
+        let buffer = Buffer.from(data);
+        let stringifiedData = buffer.toString('hex');
+        testObj.butterflyInvert = stringifiedData;
+        done();
+      });
+    });
+    before((done) => {
+      fs.readFile(`${__dirname}/butterfly-invert-test.txt`, 'hex', function(err, data) {
+        if (err) throw err;
+        testObj.invertControl = data;
+        done();
+      });
+    });
+    it('outputted test file should match newly transformed bitmap buffer', function(done){
+      expect(testObj.butterflyInvert).to.equal(testObj.invertControl);
+      done();
+    });
+  });
+});
```

Comment:
@npisciotti1 @radenska @sneakyneeks2 @darms color constructor tests are awesome!!!!  lots of em!!!